### ISSUE TITLE
Entity describer shows wrong order in yaml format

### ISSUE THREAD

**Issue by steff-petro** [drodil/backstage-plugin-toolbox#44]
I have noticed the entity describer shows the properties in the wrong order on users, groups, website and possibly other type of entities. I assume this is just a simple change in the way the data is displayed

**Comment by steff-petro** [comment]
Once I have some spare time I can look into fixing this

**Comment by drodil** [comment]
I think the describer just returns whatever catalog api returns.

**Comment by steff-petro** [comment]
@drodil Yeah that is what I assumed, thought I might change it to be the same format as the actual entity files. Because it is a bit confusing to see it in the wrong order.
